css/amp.css

```diff
@@ -250,6 +250,10 @@ i-amphtml-sizer {
   max-width: 100%;
 }
 
+.i-amphtml-intrinsic-sizer {
+  display: block !important;
+}
+
```

Comment:
There is already a line with this class selector where the rule can be added.

Reply:
+1 otherwise LGTM

Reply:
doh!